engine.code-workspace

```diff
@@ -0,0 +1,258 @@
+{
+  "folders": [
+    {
+      "path": "."
+    }
+  ],
+  "settings": {
+    "files.associations": {
+      "optional": "cpp",
```

Comment:
shouldn't these be `*.<extension>` - or are these just loose files lying about the tree.  These look like identifiers rather than files.

Reply:
These are the c++ standard library.  Like `#include <atomic>`.  Since there is no extension I think it may have issues figuring out what language it is.  I've had this set forever so I decided to share it.